include/swift/Option/Options.td

```diff
@@ -303,6 +303,8 @@ def serialize_diagnostics_path_EQ: Joined<["-"], "serialize-diagnostics-path=">,
 def color_diagnostics : Flag<["-"], "color-diagnostics">,
   Flags<[FrontendOption, DoesNotAffectIncrementalBuild]>,
   HelpText<"Print diagnostics in color">;
+def no_color_diagnostics : Flag<["-"], "no-color-diagnostics">,
+  Flags<[FrontendOption, DoesNotAffectIncrementalBuild]>;
```

Comment:
Do I need to add `HelpText` here?

Reply:
You probably should, yeah, even though it'll just be the obvious thing. No help text means the flag won't appear in `--help` output.

Reply:
Done